ci-operator/step-registry/gather/extra/jump/gather-extra-jump-commands.sh

```diff
@@ -0,0 +1,199 @@
+
+FILTER=gzip queue ${ARTIFACT_DIR}/openapi.json.gz oc --insecure-skip-tls-verify --request-timeout=5s get --raw /openapi/v2
+
+# gather nodes first in parallel since they may contain the most relevant debugging info
+while IFS= read -r i; do
+  mkdir -p ${ARTIFACT_DIR}/nodes/$i
+  queue ${ARTIFACT_DIR}/nodes/$i/heap oc --insecure-skip-tls-verify get --request-timeout=20s --raw /api/v1/nodes/$i/proxy/debug/pprof/heap
+done < ${WORKSPACE}/nodes
+
+FILTER=gzip queue ${ARTIFACT_DIR}/nodes/masters-journal.gz oc --insecure-skip-tls-verify adm node-logs --role=master --unify=false
+FILTER=gzip queue ${ARTIFACT_DIR}/nodes/masters-journal-previous.gz oc --insecure-skip-tls-verify adm node-logs --boot=-1 --role=master --unify=false
+FILTER=gzip queue ${ARTIFACT_DIR}/nodes/workers-journal.gz oc --insecure-skip-tls-verify adm node-logs --role=worker --unify=false
+FILTER=gzip queue ${ARTIFACT_DIR}/nodes/workers-journal-previous.gz oc --insecure-skip-tls-verify adm node-logs --boot=-1 --role=worker --unify=false
+
+# Snapshot iptables-save on each node for debugging possible kube-proxy issues
+oc --insecure-skip-tls-verify get --request-timeout=20s -n openshift-sdn -l app=sdn pods --template '{{ range .items }}{{ .metadata.name }}{{ "\n" }}{{ end }}' > ${WORKSPACE}/sdn-pods
+while IFS= read -r i; do
+  queue ${ARTIFACT_DIR}/network/iptables-save-$i oc --insecure-skip-tls-verify rsh --timeout=20 -n openshift-sdn -c sdn $i iptables-save -c
+done < ${WORKSPACE}/sdn-pods
+
+while IFS= read -r i; do
+  file="$( echo "$i" | cut -d ' ' -f 3 | tr -s ' ' '_' )"
+  queue ${ARTIFACT_DIR}/metrics/${file}-heap oc --insecure-skip-tls-verify exec $i -- /bin/bash -c 'oc --insecure-skip-tls-verify get --raw /debug/pprof/heap --server "https://$( hostname ):8443" --config /etc/origin/master/admin.kubeconfig'
+  queue ${ARTIFACT_DIR}/metrics/${file}-controllers-heap oc --insecure-skip-tls-verify exec $i -- /bin/bash -c 'oc --insecure-skip-tls-verify get --raw /debug/pprof/heap --server "https://$( hostname ):8444" --config /etc/origin/master/admin.kubeconfig'
+done < ${WORKSPACE}/pods-api
+
+while IFS= read -r i; do
+  file="$( echo "$i" | cut -d ' ' -f 2,3,5 | tr -s ' ' '_' )"
+  FILTER=gzip queue ${ARTIFACT_DIR}/pods/${file}.log.gz oc --insecure-skip-tls-verify logs --request-timeout=20s $i
+  FILTER=gzip queue ${ARTIFACT_DIR}/pods/${file}_previous.log.gz oc --insecure-skip-tls-verify logs --request-timeout=20s -p $i
+done < ${WORKSPACE}/containers
+
+echo "Snapshotting prometheus (may take 15s) ..."
+queue ${ARTIFACT_DIR}/metrics/prometheus.tar.gz oc --insecure-skip-tls-verify exec -n openshift-monitoring prometheus-k8s-0 -- tar cvzf - -C /prometheus .
+FILTER=gzip queue ${ARTIFACT_DIR}/metrics/prometheus-target-metadata.json.gz oc --insecure-skip-tls-verify exec -n openshift-monitoring prometheus-k8s-0 -- /bin/bash -c "curl -G http://localhost:9090/api/v1/targets/metadata --data-urlencode 'match_target={instance!=\"\"}'"
+
+wait
+
+
+EOF
```

Comment:
Are the variables in this block being translated during the `cat` call prior to being written to disk? If so, how does that affect them running on the jump host? The same thing applies to ther other `-commands.sh` in this PR.

Reply:
> Are the variables in this block being translated during the cat call prior to being written to disk?
https://stackoverflow.com/a/22698106
`'EOF'` makes it so that NO expanding will happen.

Reply:
Perfect. I wasn't aware of the functionality change when adding the quotes.